projects/plugins/jetpack/extensions/blocks/sharing-buttons/block.json

```diff
@@ -0,0 +1,32 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 1,
+	"name": "jetpack/sharing-buttons",
+	"title": "Sharing Buttons",
+	"description": "Sharing Buttons.",
+	"keywords": [ "sharing", "social" ],
+	"version": "12.5.0",
+	"textdomain": "jetpack",
+	"category": "embed",
```

Comment:
I think you should rethink the category. 

"Related posts" is also in embeds, and "Social icons" in widgets.

Neither category feels like right fit.